controls/radrichtextbox/features/selection.md

```diff
@@ -57,17 +138,91 @@ Here is an example of how to select the current word.
+	
+	    Me.radRichTextBox.Document.Selection.Clear()
+	    Dim currentParagraph As Paragraph = Me.radRichTextBox.Document.CaretPosition.GetCurrentParagraph()
+	
+	    Me.radRichTextBox.Document.Selection.AddDocumentElementToSelection(currentParagraph)
+	    Me.radRichTextBox.Delete(True)
+	End If
+{{endregion}}
+
+
+### What is Selection Range?
+
+The selection in **RadRichTextBox** consists of selection ranges. These ranges are represented by the SelectionRange class which exposes the following members:
+
+* **StartPosition**: The position determining the start of the selection range.
+* **EndPosition**: The position determining the end of the selection range.
+* **ContainsPosition()**: Determines whether the range contains a specified **DocumentPosition**.
+* **IsEmpty**: Determines whether the range contains any document elements.
+* **IsReversed**: Returns `true` when the start position is after the end position. This might be achieved as in code as well as when the user starts dragging the mouse from right to left.
+* **IsValid**: A range is considered valid when it has valid start and end positions and is not empty.
+* **RangeType**: When working with ranges, you might need to check what elements are included in a selection range. This could be achieved with the **RangeType** property of **SelectionRange**. This property is of type **SelectionRangeType** and could have one of the following values:
+
+* **Composite**: Represents selection range with mixed elements.
+* **Table**: **SelectionRange**, which contains a [Table]({%slug radrichtextbox-features-document-elements-tables%}).
+* **TableRow**: The range consists of a **TableRow**.
+* **TableCell**: **TableCell** selection range.
+
+#### **[C#] Example 3: Check the type of the elements inside the selection**
+
+{{region radrichtextbox-features-selection_4}}
+	SelectionRange selectionRange = this.radRichTextBox.Document.Selection.Ranges.First();
+	if (selectionRange.RangeType == SelectionRangeType.Table)
+	{
+	    SetTableProperties();
+	}
+{{endregion}}
+
+
+
+#### **[VB.NET] Example 3: Check the type of the elements inside the selection**
+
+{{region radrichtextbox-features-selection_5}}
+	Dim selectionRange As SelectionRange = Me.radRichTextBox.Document.Selection.Ranges.First()
+	If selectionRange.RangeType = SelectionRangeType.Table Then
+		SetTableProperties()
+	End If
+{{endregion}}
 
 
 ## Multi-Range Selection
 
-You can implement multi-range Selection by either calling multiple times the __AddSelectionStart()__ and __AddSelectionEnd()__ methods or by working with the __Ranges__ collection.
+You can implement multi-range Selection by either calling multiple times the **AddSelectionStart()** and **AddSelectionEnd()** methods of the **DocumentSelection** class or by working with the **Ranges** collection.
 
-Here is an example of selecting each "RadRichTextBox" word in the text. This example uses the first approach.
+Here is an example of how you can use the first approach.

```

Comment:
You can include this information in the capture of the code snippet. For example: "Select all occurrences of the "RadRichTextBox" word using the AddSelectionStart and AddSelectionEnd methods"